docs/tracing/docs/030-tracing-showcase.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Troubleshooting
+type: Troubleshooting
+---
+
+## Tracing Showcase
+
+### I have no micro-service or lambda configured to receive an event
```

Comment:
This should be a separate document, but not for tracing. What is the purpose of this document anyway. Why would I like to go to jeager to check event that is not consumed by anything. this is not explained. What is publish-service anyway, some random name or? Without this background info it is hard to classify what type of document it is, for sure not a troubleshooting one

Reply:
@derberg I have already left a comment to move it to event-bus documentation. This PR should solely focus on enabling developers to use tracing for issues with event-based triggers or computations.